src/window.rs

```diff
@@ -116,6 +116,26 @@ pub struct WindowAttributes {
     /// The default is `None`.
     pub max_inner_size: Option<Size>,
 
+    /// The desired position of the window. If this is `None`, some platform-specific position
+    /// will be chosen.
+    ///
+    /// The default is `None`.
+    ///
+    /// ## Platform-specific
+    ///
+    /// **macOS**: This sets the top left corner position of the window content, the window
+    /// title bar will be placed above it. If you need to precisely position the top left corner
+    /// of the whole window you have to use [`Window::set_outer_position`] after creating
+    /// the window. The window will be positioned such that it fits on screen, maintaining set
+    /// `inner_size` if any.
+    /// **Windows**: Borders might be shown outside the set position, so the position is
+    /// approximate.
```

Comment:
This should specify what kind of position it is, and to me, the wording is a bit hard to understand. I suggest

"The outer position. In other words, the top left corner of the title-bar. There may be a small gap between this position and the window due to specifics of the Window Manager"

Reply:
This is not yet addressed